### PR TITLE
LPS-160472, LPS-160473, LPS-160475, LPS-160476

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-160472
Related issue: https://issues.liferay.com/browse/LPS-160473
Related issue: https://issues.liferay.com/browse/LPS-160475
Related issue: https://issues.liferay.com/browse/LPS-160476

### CODE REVIEW

**Comment by marcelabc** [comment]
Hi @MarinhoFeliphe 
There is one integration test failing. I'm investigating, but feel comfortable to review the rest of the code :smile:

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by marcelabc** [comment]
ci:test:sf

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java [hunk 1]
```diff
@@ -524,6 +539,17 @@ else if (Objects.equals(
...
 
+	private ObjectRelationship _updateObjectRelationship(
+		long parameterObjectFieldId, String deletionType,
+		Map<Locale, String> labelMap, ObjectRelationship objectRelationship) {
+
+		objectRelationship.setParameterObjectFieldId(parameterObjectFieldId);
+		objectRelationship.setDeletionType(deletionType);
+		objectRelationship.setLabelMap(labelMap);
+
+		return objectRelationshipPersistence.update(objectRelationship);
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java:559]: Hi @marcelabc,

Could you sort alphabetically the parameters and the set methods?

Thanks!

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java [hunk 2]
```diff
@@ -772,6 +758,17 @@ private void _testSystemObjectEntryMtoMRelatedModelsProviderImpl()
 				requiredObjectRelationshipException.getMessage());
 		}
 
+		UserLocalServiceUtil.deleteUser(userIds[0]);
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java:761]: Hi @marcelabc,

Can we replace `UserLocalServiceUtil` to `_userLocalService`?

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
@marcelabc The test `CreateObject#CanAddRelationOnRelationshipField` is consistently failing since previous runs of the relevant suite. Could you check if it's passing locally (and the others as well)? Thanks :)

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:object

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:reevaluate:1317401_4946

**Comment by marcelabc** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
send it directly to BChan: https://github.com/brianchandotcom/liferay-portal/pull/122899

**Pull request closed without merging**